linux_memory_usage.py

```diff
@@ -32,9 +37,9 @@ def main():
         'Mapped', 'Active', 'Inactive'
     ]
 
-    template = "servers.{0}.system.memory.{1} {2} {3}"
+    template = args.prefix + '.{1} {2} ' + str(int(time()))
```

Comment:
It is necessary for addition.  Another good practice is to use `str.format()` multiple times by doubling `{` and `}`.  Then it would look like: `template = '{}.{{}}.{{}} {}'.format(args.prefix, int(time()))`.